src/utils/wallet.js

```diff
@@ -413,13 +413,15 @@ class Wallet {
         const accountId = this.accountId;
         const { seedPhrase, publicKey } = generateSeedPhrase();
 
-        await this.postSignedJson('/account/resendRecoveryLink', {
-            accountId,
-            method,
-            seedPhrase,
-            publicKey
-        });
-        await this.replaceAccessKey(method.publicKey, publicKey);
+        return await Promise.all([
```

Comment:
@Patrick1904 what is the reason for the change? I think there might be a problem here potentially:
1) `Promise.all` means these actions will happen in parallel.
2) It can then happen that `.postSignedJson` fails, but `.replaceAccessKey` succeeds. In that case user will get locked out of given recovery method.

Reply:
@vgrichina Good point, didn't think of that. Fixed it [here](https://github.com/near/near-wallet/pull/681/commits/013baaa8be35feb194b39e4ecf7f8ca7e8d4f345).